tests/components/zeversolar/test_init.py

```diff
@@ -27,34 +26,21 @@ async def test_async_setup_entry_fails(
         pytest.raises(ConfigEntryNotReady),
     ):
         await init.async_setup_entry(hass, config_entry)
```

Comment:
Don't use `init.async_setup_entry`, and don't check for `ConfigEntryNotReady`
```suggestion
    ):
        await hass.config_entries.async_setup(config_entry.entry_id)
```